packages/@glimmer/runtime/lib/dom/normalize.ts

```diff
@@ -43,6 +44,18 @@ export function isEmpty(value: unknown): boolean {
   return value === null || value === undefined || typeof (value as Dict).toString !== 'function';
 }
 
+let isHTML: ((value: unknown) => boolean) | undefined;
+if (typeof window !== 'undefined') {
+  let trustedTypes = (window as unknown as TrustedTypesWindow).trustedTypes;
+  if (trustedTypes?.isHTML) {
+    isHTML = trustedTypes?.isHTML.bind(trustedTypes);
+  }
+}
+
+export function isTrustedHTML(value: unknown): boolean {
```

Comment:
`function isTrustedHTML(value: unknown): value is TrustedHTML` probably?

Reply:
I really wanted to do that, and it is related to previous comment, we will need to modify whole chain of functions. I can create different PR to specifically that.

Reply:
Maybe keep it until more browsers support Trusted Types. Firefox planned to do that.

Reply:
>  Firefox planned to do that.

Do you have a reference? I'd like to update https://github.com/glimmerjs/glimmer-vm/pull/1567#issuecomment-1957989082 with the status of implementation and all that

Reply:
https://github.com/mozilla/standards-positions/issues/20#issuecomment-1853427823

Reply:
thank you!